cpp/pythagoreanTriplet.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+using namespace std;
+
+int main()
```

Comment:
The indentation style that you use is called Allman style which is like
```
function a()
{
    body
    body
}
```

Prefer instead to use the K & R style:
```
function a() {
    body
    body
}
```

the k & r style is favoured in industry.